nautobot/core/graphql/schema.py

```diff
@@ -311,7 +311,14 @@ def already_present(model):
         model = schema_type._meta.model
         type_identifier = f"{model._meta.app_label}.{model._meta.model_name}"
 
-        if type_identifier not in registry["graphql_types"].keys():
+        if type_identifier in registry["graphql_types"].keys():
```

Comment:
Calling `.keys()` is redundant since iteration of (and therefore membership tests) on a dict is the same as iterating the keys. (It's also slightly more efficient since there isn't an extra name lookup and call).

```suggestion
        if type_identifier in registry["graphql_types"]:
```